File: utils/config/s3config.py
import os
from dotenv import load_dotenv
from boto3 import client

# s3config.py
load_dotenv(dotenv_path="../../.env")

s3_client = client(
  "s3",
  region_name=os.getenv("AWS_REGION"),
  aws_access_key_id=os.getenv("AWS_ACCESS_KEY_ID"),
  aws_secret_access_key=os.getenv("AWS_SECRET_ACCESS_KEY")
)

File: utils/config/s3config.js
// s3Config.js

const { S3Client } = require("@aws-sdk/client-s3");
require("dotenv").config({ path: "../../.env" });

const s3Client = new S3Client({
  region: process.env.AWS_REGION,
  credentials: {
    accessKeyId: process.env.AWS_ACCESS_KEY_ID,
    secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
  },
});

module.exports = s3Client;

// const AWS = require("aws-sdk");
// // AWS S3 설정
// const s3 = new AWS.S3({
//   accessKeyId: process.env.AWS_ACCESS_KEY_ID,
//   secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
//   region: process.env.AWS_REGION,
// });
